index.md

```diff
@@ -5,9 +5,13 @@ layout: default
 <div class="lead pretty-links">
 StarCon is a two-day software engineering conference held at the **University of Waterloo**. Join us for two days of ten minute lighting talks about technology, computer science, programming, and the joy of learning about them!
 
+## Who's speaking? Can I apply to speak?
+
+Check out our [invited speakers](speakers/#keynotes) and [talk proposal details](speakers/#call-for-proposals)!
```

Comment:
I think it's simpler just to say that outright rn instead of linking to another page that says that.